pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -32,12 +32,15 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/probe"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 	"github.com/golang/glog"
+	flag "github.com/spf13/pflag"
 )
 
 var (
 	ErrRegistration   = errors.New("unable to register all nodes.")
 	ErrQueryIPAddress = errors.New("unable to query IP address.")
 	ErrCloudInstance  = errors.New("cloud provider doesn't support instances.")
+
+	syncNodeStatus = flag.Bool("sync_node_status", true, "Should node controler send probes to kubelets and update NodeStatus.")
```

Comment:
Done. I've added the flag to controlmanager.go, the same way as it works for sync_nodes flag.